assignments/assignment02.rb

```diff
@@ -25,11 +35,30 @@ def to_sentence(ary)
 
 # implement methods "mean", "median" on Array of numbers
 def mean(ary)
-  # your implementation here
+  # add all the numbers together
+  sum = ary.reduce {|i, acc| i + acc }
+
+  # divide by the length, convert to floating point
+  mean = sum/ary.length.to_f
+  return mean
```

Comment:
in idiomatic ruby, you don't need to declare local variable 'mean' and then explicitly return it. Instead, you can just have "sum/ary.length.to_f" be the last line of the method.